components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -2915,6 +2915,153 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
         }
     }
 
+    async createOrgAuthProvider(
+        ctx: TraceContext,
+        { entry }: GitpodServer.CreateOrgAuthProviderParams,
+    ): Promise<AuthProviderEntry> {
```

Comment:
We'll want to validate the incoming values are valid UUIDs before we use them (orgId)